src/junit.rs

```diff
@@ -63,8 +63,7 @@ fn populate(
         .time
         .as_deref()
         .or(testsuite_time)
-        .ok_or_else(|| ParserError::new_err("No time/duration found"))?
-        .parse()?;
+        .map(|t| t.parse().unwrap());
```

Comment:
as a followup, it would be good to not `unwrap` here.
we are dealing with untrusted user input, and crashing on not properly formatted numbers would be bad